packages/govuk-frontend/src/govuk/components/_all.scss

```diff
@@ -1,37 +1,2 @@
-@import "../base";
-
-@import "accordion/index";
-@import "back-link/index";
-@import "breadcrumbs/index";
-@import "button/index";
-@import "character-count/index";
-@import "checkboxes/index";
-@import "cookie-banner/index";
-@import "date-input/index";
-@import "details/index";
-@import "error-message/index";
-@import "error-summary/index";
-@import "exit-this-page/index";
-@import "fieldset/index";
-@import "file-upload/index";
-@import "footer/index";
-@import "header/index";
-@import "hint/index";
-@import "input/index";
-@import "inset-text/index";
-@import "label/index";
-@import "notification-banner/index";
-@import "pagination/index";
-@import "panel/index";
-@import "password-input/index";
-@import "phase-banner/index";
-@import "radios/index";
-@import "select/index";
-@import "skip-link/index";
-@import "summary-list/index";
-@import "table/index";
-@import "tabs/index";
-@import "tag/index";
-@import "task-list/index";
-@import "textarea/index";
-@import "warning-text/index";
+@warn "Importing using 'govuk/components/all' is deprecated. Update your import statement to import 'govuk/components/index'";
```

Comment:
What do you think about using the `warning` API for these warnings? I was thinking the warnings themselves could be different but the key could be the same.

Reply:
Potentially! My only reservation is that it might be helpful in this instance to see warnings for all instances of importing from `all`?

I could look at modifying the `_warning` mixin to make appending to `$govuk-suppressed-warnings` optional?

Reply:
I think if we wanted to make sure that you _had_ to see the warning then in that case we might as well use `@warn`. The intent of `warning` though was to allow teams to have clean build outputs if they're not equipped to handle a deprecation, however on their own heads be it if they can head the warning but choose not to. All the warnings can still have different text and different keys if we don't want to jsut turn them all off.

Reply:
I think we can still let people suppress the warnings if they want to, but they'd see all the warnings initially rather than just the one for the first thing they were trying to import?

Reply:
Aha, I'd forgotten that `warning` suppressed at the pint of delivery. Makes perfect sense to me.